mingw-w64-llvm/update-clang-from-msys2.sh

```diff
@@ -49,7 +49,8 @@ mv upstream/$pkgname/README-patches.md ./ || die "$0: failed to replace existing
 
 sed -e "s/pkgrel=[.0-9]\+\(.*\)/pkgrel=$new_pkgrel\1/" \
     -e 's/-DCMAKE_BUILD_TYPE=Release/-DCMAKE_BUILD_TYPE=MinSizeRel/' \
-    -e 's/-DLLVM_TARGETS_TO_BUILD=[^)]*/-DLLVM_TARGETS_TO_BUILD=Native/' \
+    -e 's/-DLLVM_TARGETS_TO_BUILD=[^)\\]*/-DLLVM_TARGETS_TO_BUILD=Native/' \
+    -e 's/-DLLVM_TARGETS_TO_BUILD=Native\\/-DLLVM_TARGETS_TO_BUILD=Native \\/' \
```

Comment:
I've thought about `[^)\\ ]` instead of removing and reintroducing a potential space, but decided against it to not have this break again due to a random space in a future target list.